<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电梯模拟</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/蓝/Desktop/ustc.png</normaloff>C:/Users/蓝/Desktop/ustc.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>电梯1</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="first4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>150</width>
      <height>63</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="second0">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>400</y>
      <width>150</width>
      <height>63</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="first0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>150</width>
      <height>63</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger0up">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="second2">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>260</y>
      <width>150</width>
      <height>63</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger2up">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger3up">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="first3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>150</width>
      <height>63</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="first2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>150</width>
      <height>63</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger4up">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>56</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>第四层</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>第三层</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>第二层</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>第一层</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>地下层</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="second3">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>190</y>
      <width>150</width>
      <height>63</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>等待乘客</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="second4">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>120</y>
      <width>150</width>
      <height>63</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger1up">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="first1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>150</width>
      <height>63</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="second1">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>330</y>
      <width>150</width>
      <height>63</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>电梯0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beginbtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>测试</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger4down">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger3down">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger2down">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger1down">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="passenger0down">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>430</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="timetxt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>当前时间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeedit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>测试时间：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
